OpenSim/Simulation/Model/ContactMesh.cpp

```diff
@@ -37,7 +37,7 @@ ContactMesh::ContactMesh() :
     setupProperties();
 }
 
-ContactMesh::ContactMesh(const std::string& filename, const SimTK::Vec3& location, const SimTK::Vec3& orientation, Body& body) :
+ContactMesh::ContactMesh(const std::string& filename, const SimTK::Vec3& location, const SimTK::Vec3& orientation, PhysicalFrame& body) :
```

Comment:
Perhaps change `body` to `frame` (here and on line 60) to be consistent with ContactMesh.h.